hack/update-codegen.sh

```diff
@@ -28,15 +28,17 @@ go install $(dirname $0)/../vendor/k8s.io/code-generator/cmd/deepcopy-gen
 # --output-base    because this script should also be able to run inside the vendor dir of
 #                  k8s.io/kubernetes. The output-base is needed for the generators to output into the vendor dir
 #                  instead of the $GOPATH directly. For normal projects this can be dropped.
-${CODEGEN_PKG}/generate-groups.sh "deepcopy,client,informer,lister" \
-  knative.dev/pkg/client knative.dev/pkg/apis \
-  "istio:v1alpha3 istio/authentication:v1alpha1" \
+
+# Generate our own client for istio (otherwise injection won't work)
+${CODEGEN_PKG}/generate-groups.sh "client,informer,lister" \
+  knative.dev/pkg/client istio.io/client-go/pkg/apis \
```

Comment:
Should we give the istio clients one more level of depths and place them under `knative.dev/pkg/client/istio`? The imports feel ambiguous otherwise.

Reply:
Sounds good. Can you take another look?